### PR TITLE
rework implicits in grpc sample

### PR DESCRIPTION
**Pull request by pjfanning**
* see https://github.com/apache/incubator-pekko-grpc/issues/49 for background
* this change reduces the number of implicits used

### CODE REVIEW

**Comment by sam-byng** [comment]
See #16

**Review by nvollmar — APPROVED** [review]
lgtm

**Comment by mdedetrich** [comment]
I'll look into whether this extra implicit is intentional or some kind of smell

**Review by mdedetrich — APPROVED** [review]
So this is intended, I can't remember precisely when but at some point (I believe at `ClassicActorSystem`/`ActorSystem` rework) whereby you didn't need `ActorSystem` at places where you traditionally needed to.

File: pekko-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala [hunk 1]
```diff
@@ -49,8 +49,7 @@ object ClusterClient {
 
   private def createClientStub(
     settings: ClusterClientSettings
-  )(implicit mat: Materializer, sys: ActorSystem): ClusterClientReceptionistServiceClient = {
-    implicit val ec: ExecutionContext = mat.executionContext
+  )(implicit sys: ActorSystem): ClusterClientReceptionistServiceClient = {
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply pekko-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala:52]: Only ActorSystem works. The background here is that the Akka sample uses sbt-akka-grpc 0.8.4. That is an ancient version and pre 1.0.0 so the Akka team must have changed the API before we get to sbt-akka-grpc 2.1.6 which is what sbt-pekko-grpc is based on.

**Comment by mdedetrich** [comment]
@pjfanning @sam-byng Is this PR fine to merge? I want to apply scalafmt on this repo and its best to do when there arent many PRs open.

**Comment by pjfanning** [comment]
merged

**Pull request merged**